src/components/Button/Button.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react'
-import PropTypes from 'prop-types'
+import PropTypes, { bool } from 'prop-types'
 import './Button.css'
 
-function Button({ text, onClick }) {
-  return <button className="button" onClick={onClick}>{ text }</button>
+function Button({ text, size, color, stroke, onClick }) {
+
+  let classNames = 'button'
+  classNames += ' ' + (size ? size : 'large')
+  classNames += ' ' + (color ? color : 'purple')
+  classNames += ' ' + (stroke ? 'stroke' : '')
+
+  return <button className={classNames} onClick={onClick}>{ text }</button>
 }
 
```

Comment:
I like how we have this emerging "UI Kit" ... I am thinking of developing a hidden page for development purposes where we show off the implementation of the UI Kit, which would make it clear how it should be used across the app

Reply:
yeah, sounds good